galaxy_importer/loaders.py

```diff
@@ -281,7 +283,29 @@ def _get_readme(self):
 
     def _get_metadata_description(self):
         self.log.info('Getting role description')
-        return ''
+        description = None
+        metadata_file = self._get_metadata_file(self.rel_path)
+
+        if not metadata_file:
+            self.log.warning('No role metadata found')
+            return description
+
+        with open(metadata_file) as fp:
+            role_metadata = yaml.safe_load(fp)
+        try:
+            description = role_metadata['galaxy_info']['description']
+        except KeyError:
+            self.log.warning('No role description found in role metadata')
```

Comment:
Iirc, the way the import log messages are displayed, the messages show the role that is being loaded first.
If that is true, these log messages are fine.

If it's not clear which role is being loaded, it may be useful to add some context info to the warning/error messages
(ie, 'No role description found in role metadata for role %s', self.name). But that would only be needed if that info isnt clear in the log output.

Reply:
The way log msg are displayed, user will see which role it applies to